variable/technology/technologies.yaml

```diff
@@ -460,3 +531,326 @@ Investment|Energy Supply|Liquids|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Gas:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Nuclear:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Reactance|Electricity|Transmission|Line:
+   description: Reactance. Lines need to have a reactance different from 0 in the case of a DC cable or a transportation modeling for an AC cable
+   unit: p.u.
+
+Security Factor|Electricity|Transmission|Line:
+   description: Security factor to consider approximately N-1 contingencies. NTC = TTC x SecurityFactor
+   unit: p.u.
+
+Shut Down Cost|Electricity|Coal:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut Down Cost|Electricity|Gas:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut Down Cost|Electricity|Nuclear:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut Down Cost|Electricity|Oil:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Spillage|Electricity|Energy Storage System:
+   description: Energy spillage by ESS
+   unit:  TWh
+
+Start Up Cost|Electricity|Coal:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start Up Cost|Electricity|Gas:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start Up Cost|Electricity|Nuclear:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start Up Cost|Electricity|Oil:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Storage Type|Electricity|Energy Storage System:
+   description: Storage type (daily, weekly, monthly, etc.) for energy storage system (ESS) that includes hydro, battery, etc.
+   unit: Daily/Weekly/Monthly
+
+Stored Energy Inventory|Electricity|Energy Storage System:
+   description: State of the stored enegy by ESS
+   unit:  TWh
+
+Transfer Capacity|Electricity|Transmission|Line:
```

Comment:
Is there any other capacity than transfer for a transmission line?

Reply:
Ok, I got it. Thanks.